test/api_test.js

```diff
@@ -23,6 +23,8 @@ describe('Api', () => {
   let dbWorks = false;
 
   before(async () => {
+    process.env.TASKCLUSTER_ROOT_URL = 'http://localhost:5555/';
```

Comment:
Is there a reason to save this in `process.env`?  Wouldn't a local variable also work?

Reply:
I could not detect where it comes from, but if I don't set the environment variable it doesn't work.